### PR TITLE
[kueue] Add scalability tests (PR - main)

### PR DESCRIPTION
**Pull request by trasc**
[kueue] Add scalability tests (PR - main)

### CODE REVIEW

File: config/jobs/kubernetes-sigs/kueue/kueue-presubmits-main.yaml [hunk 1]
```diff
@@ -279,3 +279,31 @@ presubmits:
           limits:
             cpu: "2"
             memory: "6Gi"
+  - name: pull-kueue-test-scalability-main
```
Comment (alculquicondor) [inline config/jobs/kubernetes-sigs/kueue/kueue-presubmits-main.yaml:282]:
```suggestion
  - name: pull-kueue-test-scheduling-perf-main
```

Given that we want to change the name of the directives #1997

Comment (trasc) [reply config/jobs/kubernetes-sigs/kueue/kueue-presubmits-main.yaml]: done

**Pull request merged**